.gitignore

```diff
@@ -0,0 +1,7 @@
+**/*~
+
+**/*.aux
+**/*.fdb_latexmk
+**/*.fls
+**/*.log
+**/*.pdf
```

Comment:
Good idea. There's a pretty well thought out `.gitignore` in the [official github gitignore repo for LaTex](https://github.com/github/gitignore/blob/master/TeX.gitignore) which I tend to pull from progressively as a work builds up in complexity / intermediate files generated.

Reply:
We can probably safely add some ignore tags for popular editors too, but that burden doesn't all have to fall on this PR -- I'm sure one of us will eventually start seeing swap files and just patch the ignore file.

Reply:
Have we decided on the citation approach yet? When we do that, there will be more stuff to add here as well.

Reply:
Yeah, something like that.